### PR TITLE
Revised mpmc_queue constructor interface - issue 118 

### PR DESCRIPTION
**Pull request by Drioastorga**
The default mode for the mpmc_queue have been removed.

### CODE REVIEW

File: include/common/mpmc_queue.h [hunk 1]
```diff
@@ -40,7 +40,7 @@ class mpmc_queue{
    public:
       using value_type = T;
 
-      mpmc_queue<T>(int _size, queue_mode active = queue_mode::blocking ):
+      mpmc_queue<T>(int _size, queue_mode active):
```
Comment (jdgarciauc3m) [inline include/common/mpmc_queue.h]: change argument name to something different from `active` which is no longer meaningful.

Perhaps mode?

**Review by jdgarciauc3m — CHANGES_REQUESTED** [review]
See code comment.

File: include/common/mpmc_queue.h (hunk 1 above)
Comment (Drioastorga) [reply include/common/mpmc_queue.h]: This argument have been renamed to "q_mode". Aditionally, the "_size" argument have been renamed to "q_size".

**Review by jdgarciauc3m — CHANGES_REQUESTED** [review]
This does not compile. Please review.

**Comment by Drioastorga** [comment]
Solved. It was a mistake in the last commit and push.

**Review by jdgarciauc3m — APPROVED** [review]
OK

**Comment by jdgarciauc3m** [comment]
Solves issue #118

**Pull request merged**